styles.css

```diff
@@ -147,7 +152,24 @@ main {
   margin-left: 1%;
   flex-direction: column;
   box-shadow:  5px 5px 10px #b3b3b3,
-             -5px -5px 10px #ffffff;
+  -5px -5px 10px #ffffff;
+}
+
+.card-div {
```

Comment:
You need to have a `display: flex` in order for flexbox properties to be used. LIke `justify-content`

Reply:
Thank you! I removed it as it was not needed.